<launch>
  <node name="tree_cluster3d" pkg="dbscan" type="dbscan3d" output= "screen" >
    <param name="scan_number_360" type="double" value="1080.0"/>
    <param name="scan_range" type="double" value="8.0"/>
    <param name="tree_point" type="string" value="/tree_pt"/>
    <param name="scan_topic_name" type="string" value="/velodyne_points
"/>

    <param name="EPS" type="double" value="0.1"/>
    <param name="MinPts" type="int" value="5"/>
    <param name="tree_residual" type="double" value="0.01"/>
    <param name="tree_radius_max" type="double" value="0.18"/>
    <param name="tree_radius_min" type="double" value="0.05"/>
    <param name="min_cluster" type="double" value="20"/>

    <param name="distance_max" type="double" value="30.0"/>
    <param name="distance_min" type="double" value="0.0"/>
    <param name="height_max" type="double" value="2.0"/>
    <param name="height_min" type="double" value="0.0"/>
  </node>
  
</launch>
